sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -293,6 +297,16 @@ def begin_update(
         update_param.container_registry = container_registry
         update_param.application_insights = application_insights
 
+        # Only the key uri property of customer_managed_key can be updated.
+        # Check if user is updating CMK key uri, if so, add to update_param
+        if workspace.customer_managed_key is not None and workspace.customer_managed_key.key_uri is not None:
+            customer_managed_key_uri = kwargs.get("customer_managed_key_uri", workspace.customer_managed_key.key_uri)
```

Comment:
Add ref doc for this new arg "customer_managed_key_uri"?

Reply:
We are not going to expose a customer parameter for this field since it will not be used as often. Going to remove that logic and directly set the value to workspace.customer_managed_key.key_uri instead.